src/app/static/src/js/components/runReport/gitUpdateReports.vue

```diff
@@ -109,7 +110,8 @@
             },
             updateReports() {
                 this.reports = [];
-                const query = this.reportMetadata.git_supported ? `?branch=${this.selectedBranch}&commit=${this.selectedCommitId}` : '';
+                const showAllParam = this.showAllReports ? "&show_all=true" : "";
+                const query = this.reportMetadata.git_supported ? `?branch=${this.selectedBranch}&commit=${this.selectedCommitId}${showAllParam}` : '';
```

Comment:
We don't need to worry about passing `show_all` if git is not supported, since in that case we only get reports for the master branch, which returns all reports by default.